cartographer/io/probability_grid_points_processor.h

```diff
@@ -40,14 +40,15 @@ class ProbabilityGridPointsProcessor : public PointsProcessor {
   constexpr static const char* kConfigurationFileActionName =
       "write_probability_grid";
   enum class DrawTrajectories { kNo, kYes };
+  enum class OutputType { kPng, kPb };
   ProbabilityGridPointsProcessor(
       double resolution,
       const mapping::proto::ProbabilityGridRangeDataInserterOptions2D&
           probability_grid_range_data_inserter_options,
       const DrawTrajectories& draw_trajectories,
       std::unique_ptr<FileWriter> file_writer,
       const std::vector<mapping::proto::Trajectory>& trajectorios,
-      PointsProcessor* next);
+      const OutputType& output_type, PointsProcessor* next);
```

Comment:
Should this argument be moved right after `draw_trajectories`?

Reply:
yes, I fixed it!